internal/tools/scripts/schema-diff/yaml/schema-diff.j2

```diff
@@ -0,0 +1,18 @@
+{{- template.set_file_name("schema-diff.yaml") }}
+{% if ctx.registry_attributes | length > 0 %}
+    all:
+      changes:
+        - rename_attributes:
+            attribute_map:
+{% for attr in ctx.registry_attributes %}
+              {{attr.old_name}}: {{attr.new_name}}
+{% endfor %}
+{% endif %}
+{% if ctx.metrics | length > 0 %}
+    metrics:
+      changes:
+        - rename_metrics:
+{% for metric in ctx.metrics %}
+              {{metric.old_name}}: {{metric.new_name}}
```

Comment:
apparently this change is blocked on https://github.com/open-telemetry/weaver/issues/776 - old_name is not a name, but id

Reply:
fixed in weaver 0.15.3